cfme/fixtures/ansible_tower.py

```diff
@@ -0,0 +1,126 @@
+import fauxfactory
+import pytest
+
+from cfme.rest.gen_data import _creating_skeleton
+from cfme.utils.blockers import GH
+
+
+def ansible_tower_dialog_rest(request, appliance):
+    """Creates service dialog using REST API."""
+    uid = fauxfactory.gen_alphanumeric()
+    data = {
```

Comment:
@nachandr Thoughts on putting this in a yaml file in `integration_tests.data` directory? That way this file will not be overcrowded with this dictionary.